### ISSUE TITLE
Errors while running this code

### ISSUE THREAD

**Issue by CyberSatish** [wnlUc3m/AZTEC#1]
while i am running this code, i got error of "num_iters , load_input, load_output variables are not defined".  Is it related to input data? If so where can i get input dataset? If it is not available hoe can i create my own dataset ( I mean what could be the format of input dataset)?

Please help me
Thanks in Advance

**Comment by harleyxu-xhl** [comment]
Yes, there is no dataset in this notebook. I also want to know how to get the datasets? Or,could you provide the data format for us so that we can make one by ourselves.
